lib/xdr/contract/spec/sc_spec_type_map.ex

```diff
@@ -0,0 +1,59 @@
+defmodule StellarBase.XDR.SCSpecTypeMap do
+  @moduledoc """
+  Representation of Stellar `SCSpecTypeMap` type.
+  """
+
+  alias StellarBase.XDR.SCSpecTypeDef
+
+  @behaviour XDR.Declaration
+
+  @struct_spec XDR.Struct.new(keyType: SCSpecTypeDef, valueType: SCSpecTypeDef)
+
+  @type keyType :: SCSpecTypeDef.t()
+  @type valueType :: SCSpecTypeDef.t()
+
+  @type t :: %__MODULE__{keyType: keyType(), valueType: valueType()}
+
+  defstruct [:keyType, :valueType]
```

Comment:
Follow the naming conventions for Elixir, i.e, camel case for variable names. Several modules with the same issue, please fix!